ui/src/lib/RegexBuilder.js

```diff
@@ -39,11 +39,15 @@ export default class RegexBuilder {
       .reduce(
         (acc, [key, value]) => ({
           ...acc,
-          [key]: false,
+          [key]: false
         }),
         {}
       );
   }
 
-  getFlag() {}
+  getFlag() {
+    return Object.keys(this.flags)
```

Comment:
just to be sure that we get all values for `REGEX_FLAGS`, what about start this with `Object.keys(REGEX_FLAGS)`?